### PR TITLE
Added support for Alibaba Cloud DNS

### PR DESCRIPTION
**Pull request by loganhz**

### CODE REVIEW

**Comment by yasker** [comment]
@loganhz 

Can you make PR to `rancher-catalog` instead? Because we're supporting it.

File: infra-templates/alidns/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# notemplating
+.catalog:
+  name: "Alibaba Cloud DNS"
+  version: "v0.1.0-rancher1"
```
Comment (deniseschannon) [inline infra-templates/alidns/0/rancher-compose.yml]: please remove the `-rancher1`

File: infra-templates/alidns/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  description: "Rancher External DNS service powered by Alibaba Cloud"
+  minimum_rancher_version: v1.6.0-rc1
```
Comment (deniseschannon) [inline infra-templates/alidns/0/rancher-compose.yml]: please update this to v1.6.8-rc1

File: infra-templates/alidns/config.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+name: Alibaba Cloud DNS
+description: |
+  Rancher External DNS service powered by Alibaba Cloud
+version:  v0.1.0-rancher1
```
Comment (deniseschannon) [inline infra-templates/alidns/config.yml]: please remove the `-rancher1`

**Comment by loganhz** [comment]
updated
@deniseschannon thanks for your review.

**Comment by yasker** [comment]
@Tracy1994 Can you retest this PR for Aliyun DNS?

**Comment by Tracy1994** [comment]
This PR has been tested and passed.
Test case: https://github.com/Tracy1994/rancher-test_case/blob/master/alibaba_clound_dns_testcase.xlsx

**Pull request merged**